docker-compose.extended.yml

```diff
@@ -5,16 +5,13 @@ services:
             - "/var/run/docker.sock:/var/run/docker.sock"
         ports:
             - 8080:8080
-        image: functions/gateway:0.6.2 
+        image: functions/gateway:0.6.3 
         networks:
             - functions
         environment:
             dnsrr: "true"  # Temporarily use dnsrr in place of VIP while issue persists on PWD
             faas_nats_address: "nats"
             faas_nats_port: 4222
-        deploy:
```

Comment:
@developius we had `placement` for the gateway twice which is how you managed to schedule it on a 'worker' node.

Reply:
Missed this. Makes sense